ee/modules/160-multitenancy-manager/templates/user-resources/user-resources-templates.yaml

```
@@ -119,9 +119,9 @@ spec:
   accessLevel: {{ $values.role }}
```

There is no point in keeping two terms. It will only confuse users.

```suggestion
  accessLevel: {{ $values.accessLevel }}
```